kit/inputs/src/UIMaterialTextView/UIMaterialTextView.tsx

```diff
@@ -381,10 +390,14 @@ const UIMaterialTextViewSimple = React.forwardRef<
         props,
         onHeightChange,
     );
-    const clear = React.useCallback(() => {
-        clearInput();
+    const clearInput = React.useCallback(() => {
+        clearProp();
         resetInputHeight();
-    }, [clearInput, resetInputHeight]);
+    }, [clearProp, resetInputHeight]);
+
+    const clear = React.useMemo(() => {
+        return success ? undefined : () => clearInput();
```

Comment:
So what's the point of the last commit if it does the same? btw `() => clearInput()` can be safely changed to `clearInput`

Reply:
@savelichalex This is @sertony idea

Reply:
@sertony I didn't get the idea